scrapy/mail.py

```diff
@@ -129,12 +129,29 @@ def _sendmail(self, to_addrs, msg):
         # Import twisted.mail here because it is not available in python3
         from twisted.internet import reactor
         from twisted.mail.smtp import ESMTPSenderFactory
-        msg = BytesIO(msg)
+        from twisted import version as twisted_version
```

Comment:
why local imports?

Reply:
I did it to conform to the code that was already there. You think it's better to move them all up?

Reply:
Didn't know that. Will remove the comment as well